src/components/editTestCase/EditTestCase.tsx

```diff
@@ -643,15 +645,22 @@ const EditTestCase = () => {
   };
 
   const severityOfValidationErrors = (validationErrors) => {
+    const hasSeverityProperty = validationErrors?.filter(
+      (validationError) => !validationError.hasOwnProperty("severity")
+    ).length;
```

Comment:
A little bit confusing here. name of the property is `hasSeverityProperty` but you are checking for the negation of it.

Reply:
the reason for using this is there are few situations like when the madie-fhir-service is not running in this case the HAPI Operation Outcome doesn't have any severity property so I am filtering that record so that if it there are any such outcomes I can directly set to error

Reply:
Oops, I read the question wrong I'll change the name of it to something relevant. thanks for the suggestion Ashok